modules/_node-scripts/esbuild/getEntryPoint.mjs

```diff
@@ -0,0 +1,21 @@
+import getFlatName from '../util/getFlatName.mjs';
+import getExportBridgePath from './getExportBridgePath.mjs';
+
+export default function getEntryPoint(moduleName) {
+	let entryPoint;
+
+	if (moduleName.endsWith('.css')) {
```

Comment:
Do we need to check scss?

Reply:
Same answer as in https://github.com/liferay-frontend/liferay-portal/pull/4125#discussion_r1597997600